conformance/tests/gateway-invalid-tls-backend-configuration.go

```diff
@@ -0,0 +1,95 @@
+/*
+Copyright 2025 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package tests
+
+import (
+	"testing"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+
+	gatewayv1 "sigs.k8s.io/gateway-api/apis/v1"
+	"sigs.k8s.io/gateway-api/conformance/utils/kubernetes"
+	"sigs.k8s.io/gateway-api/conformance/utils/suite"
+	"sigs.k8s.io/gateway-api/pkg/features"
+)
+
+func init() {
+	ConformanceTests = append(ConformanceTests, GatewayInvalidTLSBackendConfiguration)
+}
+
+var GatewayInvalidTLSBackendConfiguration = suite.ConformanceTest{
+	ShortName:   "GatewayInvalidTLSBackendConfiguration",
+	Description: "A Gateway should have ResolvedRefs condition set false if the Gateway has an invalid backend TLS configuration",
+	Features: []features.FeatureName{
+		features.SupportGateway,
```

Comment:
it should also support `features.SupportBackendTLSPolicy` and HTTP route because for now it can be used only with Services attached to HTTPRoutes right?